package.json

```diff
@@ -80,6 +81,7 @@
     "react-intersection-observer": "^8.31.0",
     "react-lottie-player": "^1.4.0",
     "react-number-format": "^4.4.4",
+    "react-popper": "^2.2.5",
```

Comment:
Let's remove those deps, they're not part of this PR, so they shouldn't be here.